etc/ci.sh

```diff
@@ -46,10 +48,23 @@ make
 # return to base directory
 popd
 
-# create dir for coverage results
-COVDIR=coverage
-mkdir -p $COVDIR
-
+# When we want to record coverage, we enable the user preference
+# ReproducibleBehaviour, which stops code depending on Runtime(),
+# which give different results on different test executions.
+if [ -z "${NO_COVERAGE+x}" ]
+then
```

Comment:
If we never want to do a no-coverage run on manuals (for example), that would be fine. That also simplifies things for now.